src/notification/notification-content.interface.ts

```diff
@@ -2,12 +2,14 @@ export interface NotificationContent {
 	type: string;
 	title: string;
 	target?: string;
+	actionText?: string;
 	duration?: number;
 	smart?: boolean;
 	closeLabel?: any;
 	message?: string;
 	showClose?: boolean;
 	lowContrast?: boolean;
+	showAction?: boolean;
```

Comment:
Would it make more sense to have it be `actions: NotificationAction[]` where `NotificationAction` looks like

```typescript
interface NotificationAction {
    text: string;
    click: Subject<ClickEvent> | (event: ClickEvent) => any;
}
```

So that a user could define multiple actions?